modules/apps/fragment/fragment-impl/package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "fragment-impl",
+	"scripts": {
+		"build": "liferay-npm-scripts build",
```

Comment:
We probably don't need this?

Reply:
Replied https://github.com/liferay-frontend/liferay-portal/pull/268#pullrequestreview-472312434